### ISSUE TITLE
Website claims mixed "not logged in" / "logged in" when opened in specific container

### ISSUE THREAD

**Issue by lissyx** [mozilla/testpilot-containers#525]
STR:
 1. Open https://moncompte.sncf.com in « Personnal » and identify yourself
 2. Top shows menu "Mon compte" followed by your name and a disconnect link in the menu
 3. Navigating to an item to place an order cannot be completed, the website claims that "You must be logged in".

Do the same outside of any container, and you can place the order.

**Comment by lissyx** [comment]
![capture d ecran de 2017-05-22 16-34-34](https://cloud.githubusercontent.com/assets/1645737/26313903/9eb47cba-3f0c-11e7-9771-e6d8ebcd65c0.png)
![capture d ecran de 2017-05-22 16-34-31](https://cloud.githubusercontent.com/assets/1645737/26313904/9eb811fe-3f0c-11e7-8022-43d188496b60.png)

**Comment by lissyx** [comment]
The content of the online shop lives within an iframe, at https://www.boutique-programme-voyageur.sncf.com/minisite/Framework?... .
![capture d ecran de 2017-05-22 16-37-48](https://cloud.githubusercontent.com/assets/1645737/26314055/0b072156-3f0d-11e7-84de-1efc86f4fa69.png)

**Comment by jonathanKingston** [comment]
@kjozwiak is this something you would be able to look into? I'm a little worried this is a containment issue. It could also perhaps be the cookie issue which just landed in nightly (potentially even that patch breaking it, @lissyx is using Nightly it seems).

**Comment by lissyx** [comment]
My nightly was older than the one from today.

**Comment by lissyx** [comment]
toolkit.telemetry.previousBuildID=20170522100359

**Comment by lissyx** [comment]
Sadly, I cannot reproduce that under mozregression. I already tried disabling uBlock of course :)

**Comment by lissyx** [comment]
In safe mode, all addons disabled, I still reproduce the issue.

**Comment by jonathanKingston** [comment]
@kjozwiak I think this might need your attention. @lissyx is "gerard-majax" on irc if you need help etc. I spent some time yesterday speaking through obvious things here. The main issue is that getting an account on this site requires a season ticket rail card which is likely not possible (I dunno if they might be persuaded to give us test accounts perhaps).

It sounded to me like this was an intermittent cookie clearing issue as using mozregression the issue didn't always happen and didn't seem related to: https://bugzilla.mozilla.org/show_bug.cgi?id=1334587

**Comment by lissyx** [comment]
Some news. I did reproduce again the issue with today's nightly. I took a chance, and cleaned ALL cookies, while filtering on "sncf.com". So it included *.sncf.com and voyages-sncf.com. As a result, it does work now in BOTH contexts. I'm going to close this as WORKSFORME, since there is nothing really actionnable. To date, this is not the first time in a couple of weeks/months that I have cookies issues similar to this one :(
